src/signals/incident-management/components/FieldControlWrapper/index.js

```diff
@@ -45,6 +47,7 @@ class FieldControlWrapper extends Component {
     return null
   }
 
+  // Bij een update, update values and valideer
   componentDidUpdate(prevProps) {
```

Comment:
moeten we hier niet vanaf? componentDidUpdate is toch (bijna) deprecated?

Reply:
we moeten er wel vanaf. dat gebeurt vanzelf als we alles ombouwen naar react hook form.Dan zijn die wrapper componenten niet meer nodig